src/api/index.ts

```diff
@@ -69,7 +69,9 @@ const api = async (customConfig: any) => {
         data: customConfig.data,
         params: customConfig.params
     }
-    const baseURL = process.env.VUE_APP_BASE_URL;
+
+    let baseURL = store.getters['user/getInstanceUrl'];
+    baseURL = baseURL.match(/^https?:/) ? baseURL : 'https://' + baseURL + '.hotwax.io/api/';
```

Comment:
What if I have added an URL like: `https://dev-hc` or `dev-hc.india.co`?

Reply:
Improved the code to change the base URL according to the output.
Thank You for reviewing my code.

Reply:
Use http

Reply:
Used http instead of https.
Thank You for reviewing my code.